src/ngraph/runtime/cpu/cpu_external_function.hpp

```diff
@@ -78,12 +80,19 @@ namespace ngraph
                 std::string Description;
                 std::vector<std::string> Outputs;
                 std::vector<std::string> Inputs;
+                std::vector<std::pair<size_t, ngraph::Shape>> Outputs_tensor_attrs;
```

Comment:
Last comment. You might want to capture element_type as well here so you could dump tensors only for float element types instead of the current implementation which seems to assume all tensors are float types.

Reply:
@jbobba, I captured it. I have two options to solve this problem
(*my assumption that you are talking about the same type for one run and NOT mixed types for one iteration?!)
1) I can add switch/case with input "get_type_enum" in "dump_one_kernel".
2) I can template all the debugger class, but here I need to know from the beginning of built function what type tensors will be(same type for all) (if I know the type it will be better solution)

Reply:
yes, same type for one run. (1) seems sufficient.